bitnami/kafka/values.yaml

```diff
@@ -517,29 +517,34 @@ resources:
 ## ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
 ## @param livenessProbe.enabled Enable livenessProbe
 ## @param livenessProbe.initialDelaySeconds Initial delay seconds for livenessProbe
+## @param livenessProbe.periodSeconds Period seconds for livenessProbe
 ## @param livenessProbe.timeoutSeconds Timeout seconds for livenessProbe
+## @param livenessProbe.failureThreshold Failure threshold for livenessProbe
+## @param livenessProbe.successThreshold Success threshold for livenessProbe
 ##
 livenessProbe:
   enabled: true
   initialDelaySeconds: 10
   timeoutSeconds: 5
-  # failureThreshold: 3
-  # periodSeconds: 10
-  # successThreshold: 1
+  failureThreshold: ""
+  periodSeconds: ""
+  successThreshold: ""
```

Comment:
@carrodher initially after a chat with @migruiz4 I opted for setting this to `""` and the installation worked just fine as opposed to Logstash. Now that we have set these values for Logstash in #7024 7045 we should do the same here, right?

Reply:
Yes, in order to unify all the charts I think the best option is to set default values as we are doing in the rest.
Talking about logstash, I saw in the other PR that you set default values for the main container but not for the metrics one, just in case you want to add those changes here in order to have all the probes with default values

Reply:
Done! Thanks again Carlos